model/things/src/main/java/org/eclipse/ditto/model/things/ThingBuilder.java

```diff
@@ -992,7 +1054,7 @@ default FromCopy setFeature(final Feature feature) {
         /**
          * Sets the given Feature to this builder. A previously set Feature with the same ID is replaced.
          *
-         * @param existingFeaturesPredicate a predicate to decide whether the given features are set. The predicate
+         * @param existingFeaturesPredicate a predicate to decide whether the given features exist. The predicate
```

Comment:
Descriptions of predicates are not yet correct (in other classes, too).
The given predicate determines whether the provided arguments are set to the builder (depending on the Features in this case).
This should be transported by the Javadoc comment.

Reply:
Changed in commit: https://github.com/eclipse/ditto/pull/865/commits/41a3a6d494a67cd4e4a76e7103d147d8d0e35e05
Can you have a check?